README.md

```diff
@@ -329,32 +329,33 @@ lightbox opened from clicking on a thumbnail, for example).
-responsive  | `false`  | Resize according to the parent element (experimental)
-speed       | `false`  | Show the speed controls in the preferences menu and enable playback rate API (available to PRO and Business accounts).
-texttrack   |          | Turn captions/subtitles on for a specific language by default. If you enter a language preference that hasn't yet been uploaded for your particular video, the text track parameter will be ignored, and your embedded video may load with CC or subtitles disabled by default. Supports lowercase language code (such as: `fr`, `es`, `de`, `en`). You can find a full list of popular language codes [here](https://www.andiamo.co.uk/resources/iso-language-codes/).
-title       | `true`   | Show the title on the video.
-transparent | `true`   | The responsive player and transparent background are enabled by default, to disable set this parameter to `false`.
-width       |          | The exact width of the video. Defaults to the width of the largest available version of the video.
+option            | default  | description
+----------------- | -------- | -----------
+id _or_ url       |          | **Required.** Either the id or the URL of the video. Note that if the video [privacy settings](https://vimeo.zendesk.com/hc/en-us/articles/224817847-Privacy-settings-overview) are "Private", instead of an id, a URL that includes the `h` parameter must be provided.
+autopause         | `true`   | Pause this video automatically when another one plays.
+autoplay          | `false`  | Automatically start playback of the video. Note that this won’t work on some devices.
+background        | `false`  | Enable the player's background mode which hides the controls, autoplays and loops the video (available to  Plus, PRO, or Business members).
+byline            | `true`   | Show the byline on the video.
+color             | `00adef` | Specify the color of the video controls. Colors may be overridden by the embed settings of the video.
+controls          | `true`   | This parameter will hide all elements in the player (play bar, sharing buttons, etc) for a chromeless experience. ⚠️Warning: When using this parameter, the play bar and UI will be hidden. To start playback for your viewers, you'll need to either enable autoplay or use our player SDK to start and control playback. (available to Plus, PRO, or Business members)
+dnt               | `false`  | Block the player from tracking any session data, including cookies.
+height            |          | The exact height of the video. Defaults to the height of the largest available version of the video.
+interactiveparams |          | For interactive videos only that utilize live elements, this will contain key/value pairs for dynamic values to be passed into the live element. Ex: `[key1=value1,key2=value2]` 
```

Comment:
```suggestion
interactiveparams |          | Key-value pairs representing dynamic parameters that are utilized on interactive videos with live elements. Ex: `[key1=value1,key2=value2]`
```

We should also ask Player Tools / Alex C whether there's an article or documentation that we can link to from here that can provide more context.

Reply:
Per Conrad, they don't have front-facing documentation for this feature yet, and he recommended we don't link to the older Wirewax docs, so we may have to circle back at a later date to add a doc link.